24-bit truecolor support =＾● ⋏ ●＾=

vaz opened the pull request:
Adding maybe like 16 million or so colors just in case you need them.

This branch depends on a patch to `paint` (see janlelis/paint#15) which I dunno if they actually want because I just made the PR right now. I have an alternate branch [here](https://github.com/vaz/lolcat/tree/truecolor-alt) that bypasses `paint` for the truecolor though so don't even worry.

It's pretty much a lot slower printing in truecolor than it already was, but on the other hand just look at it srsly

![lol](https://cl.ly/1d1w3b3S0i0U/download/Image%202016-12-17%20at%201.43.05%20%E5%8D%88%E5%89%8D.png)

(Oh also --invert option yeah?)

m-o-e commented:
Very good work!

Let's give @janlelis till the end of January to accept your truecolor-PR
(holidays and all), otherwise I'll look into merging the non-Paint version.

vaz commented:
Sounds good. I just updated this branch because I changed something in the Paint PR. But it looks like they haven't been active for a year anyway. I guess we'll see!

m-o-e commented:
@vaz Thank you!
